schema/divisions/boundary.yaml

```diff
@@ -30,16 +30,16 @@ properties:     # JSON Schema: Top-level object properties.
 
           - maritime    # All the boundary geometry extends beyond the
                         # coastline of both associated divisions.
-      divisions:
+      division_ids:
         description:
-          The two divisions to the left and right, respectively, of the
-          boundary line. The left- and right-hand sides of the boundary are
-          considered from the perspective of a person standing on the
-          line facing in the direction in which the geometry is oriented,
-          i.e. facing toward the end of the line.
+          Identifies the two divisions to the left and right, respectively, of
+          the boundary line. The left- and right-hand sides of the boundary are
+          considered from the perspective of a person standing on the line
+          facing in the direction in which the geometry is oriented, i.e. facing
+          toward the end of the line.
           
-          The first array element represents the left division. The second
-          element represents the right division.
+          The first array element is the Overture ID of the left division. The
+          second element is the Overture ID of the right division.
```

Comment:
Overture ID or GERS id? Since technically division GERS ids will go there.

Reply:
My understanding is we're aiming to rename "GERS ID" to "Overture ID" to reduce cognitive load. Same concept, same stable ID, different name.

@jenningsanderson @danabauer Do I have this right?

Reply:
We're in limbo until the Steering Committee weighs in. One possible (and likely) outcome is that we'll refer to the `id` as "Overture ID" and the reference system as GERS.

Reply:
I agree that you should use "Overture ID"